### PR TITLE
Add transforms, features, and Claudius PR

### PR DESCRIPTION
**Pull request by KonstantinWilleke**
PR that has accumulated a bunch of small features.

- Scale transformation for the NeuroNormalizer in transforms.py
- solving a bug when normalizing the behavior states, and giving the eye_position the right name (pupil_center)
- adding a 5x5 laplace filter for the Laplace regularizer
- initializing a FileTree Dataset without behavior
- adding a new regularizer to the gaussian readout, that just returns the standard deviations of positions learned.

- Add the deterministic gaussian readout of Claudius

### CODE REVIEW

**Comment by KonstantinWilleke** [comment]
This PR is a replacement for https://github.com/sinzlab/neuralpredictors/pull/89.
All issues that were mentioned there are addressed.

File: neuralpredictors/regularizers.py [hunk 1]
```diff
@@ -17,6 +17,31 @@ def laplace():
     return np.array([[0, -1, 0], [-1, 4, -1], [0, -1, 0]]).astype(np.float32)[None, None, ...]
 
 
+def laplace5x5():
```
Comment (eywalker) [inline neuralpredictors/regularizers.py:20]: Where did the definition of these filters come from? It actually appears to LoG, not Laplace filters.

File: neuralpredictors/regularizers.py [hunk 2]
```diff
@@ -17,6 +17,31 @@ def laplace():
...
+    return np.array([[0, 0, 1, 0, 0],
+                     [0, 1, 2, 1, 0],
+                     [1, 2, -16, 2, 1],
+                     [0, 1, 2, 1, 0],
+                     [0, 0, 1, 0, 0]]).astype(np.float32)[None, None, ...]
+
+
+def laplace7x7():
+    """
+    Returns a 5x5 laplace filter.
```
Comment (eywalker) [inline neuralpredictors/regularizers.py:34]: update this to say 7x7

File: neuralpredictors/data/transforms.py [hunk 3]
```diff
@@ -298,24 +299,24 @@ def __init__(self, data, stats_source="all", exclude=None):
...
-            self._eye_std = np.array(
-                data.statistics["eye_position"][stats_source]["std"]
-            )
-            transforms["eye_position"] = lambda x: (x - self._eye_mean) / self._eye_std
-            itransforms["eye_position"] = lambda x: x * self._eye_std + self._eye_mean
 
+        # -- behavior
+        transforms['behavior'] = lambda x: x
+
+        if "pupil_center" in data.data_keys:
```
Comment (eywalker) [inline neuralpredictors/data/transforms.py:309]: make this work with either "pupil_center" or "eye_position" and not just overwrite. Static datasets use "pupil_center" and movie datasets are using "eye_position"

File: neuralpredictors/data/transforms.py [hunk 4]
```diff
@@ -298,24 +299,24 @@ def __init__(self, data, stats_source="all", exclude=None):
...
+            self._eye_std = np.array(data.statistics["pupil_center"][stats_source]["std"])
+            transforms["pupil_center"] = lambda x: (x - self._eye_mean) / self._eye_std
+            itransforms["pupil_center"] = lambda x: x * self._eye_std + self._eye_mean
+
+        if "behavior" in data.data_keys:
             s = np.array(data.statistics["behavior"][stats_source]["std"])
 
             # TODO: same as above - consider other baselines
-            threshold = 0.01 * s.mean()
+            threshold = 0.01 * s
```
Comment (eywalker) [inline neuralpredictors/data/transforms.py:319]: this definition doesn't make sense as `s > 0.01 * s` is true all the time provided that `s` is a positive array (which it is as it's std values), so either completely get rid of this thresholding behavior, or come up with a meaningful thresholding scheme.

File: neuralpredictors/data/transforms.py [hunk 5]
```diff
@@ -386,6 +389,8 @@ def __call__(self, x):
             "responses": self.transforms["responses"](key_vals["responses"]),
```
Comment (eywalker) [inline neuralpredictors/data/transforms.py:389]: why apply transforms given that `transforms` for `responses`, `behavior` and `pupile_center` are set as no-op. Do you expect these values to be overridden?

**Review by eywalker — CHANGES_REQUESTED** [review]

**Review by eywalker — APPROVED** [review]

**Pull request merged**